packages/lodestar/src/network/reqResp.ts

```diff
@@ -206,7 +206,6 @@ export class ReqResp extends (EventEmitter as IReqEventEmitterClass) implements
           }
           return;
         }
-        yield* getResponse(peerId, method, encoding);
```

Comment:
we never reach this statement because we always yield something in `eth2RequestDecode`